node_3d.gd

```diff
@@ -0,0 +1,18 @@
+extends Node3D
+
+var clicks = 0 #Varible Clicks que se va a usar en gran parte de los Nodos
+var segs = 0 #Varible Segundos para usar con AutoClicker
+var x = 0.25 #Variable de Clicks Extras (por el autoclicker) WIP
+var segs2 = 0 #Variable que se va a ir comparando con los segundos
+
+# Called when the node enters the scene tree for the first time.
+#func _ready():
+	#$Label.text = str(clicks) #Click esta en int y se escribe a str par que se vea como texo
+
+
+func _process(delta):
```

Comment:
El parametro delta no se usa en toda la función, no tiene por qué estar ahí. Plus, la func "_process" se ejecuta una vez por frame, generando que quien tenga más FPS traiga una ventaja. Si necesitas más info de esto pregunta por el Discord

Reply:
The function signature doesn't match the parent. Parent signature is "_process(float) -> void".